_sass/_opensearch.scss

```diff
@@ -193,17 +194,20 @@ $max-container: 1440px;
     }
 }
 
-.img-fluid {
+img {
```

Comment:
@kolchfa-aws This selector is a type selector that will target all images everywhere regardless of location in the docuement. If the goal is to affect images within the content without needing to apply the Kramdown extension for applying the `.img-fluid` class then I might suggest that the selector be more deliberate, and nested within the selector for the parent of the content as to prevent unforseen consequences with affecting images anywhere on the site that might appear now, or in the future without needing to override if at all necessary. For example Blog post content is interpolated into the `#content-main` element.

Reply:
Thanks, @astephanus! Let me correct that.